### PR TITLE
fix : Code accessible when submission clicked

### LINKED ISSUE #10: View the submission also along with problem
**Describe the feature you'd like**

When a submission is pressed in the "recent submissions" it normally directs to the problem question.
Now can implement also to show the problem submission code along with the problem question.
Like a dialog box to choose question or submission code.

**Additional context**

It can be done by using the submission id retrieved from the API.

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by abhinandhari**
Fixes #10 

**Changes:**
When a submission is clicked, the code can be seen.

[Add here what changes were made in this issue and if possible provide links.]

Please make sure these boxes are checked before submitting your pull request - thank you!

- [x] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into a single commit by squashing them.

Please add screenshots/GIFs here if there are any changes to the UI below.
codeCanBeSeen 1

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,29 +1,29 @@
-<component name="ProjectCodeStyleConfiguration">
```
Comment (immadisairaj) [inline .idea/codeStyles/Project.xml:1]: can you please ignore .idea/ files while sending a PR

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionAdapter.java [hunk 2]
```diff
@@ -100,8 +101,10 @@ public void onClick(View v) {
                 if (submissionList.get(position).getProblem().getProblemsetName() == null) {
                     if (submissionList.get(position).getProblem().getContestId() >= 100000)
                         url += "gym/" + submissionList.get(position).getProblem().getContestId() + "/problem/" + submissionList.get(position).getProblem().getIndex();
-                    else
-                        url += "problemset/problem/" + submissionList.get(position).getProblem().getContestId() + "/" + submissionList.get(position).getProblem().getIndex();
+                    else {
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionAdapter.java]: can there be a dialog box where the user can either select the problem question or the submission s/he made

**Review by immadisairaj — COMMENTED** [review]
can there be a dialog box where the user can either select the problem question or the submission s/he made

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (abhinandhari) [reply .idea/codeStyles/Project.xml:1]: Ok, will do.

**Pull request closed without merging**